File: src/plugin/npc/SlayerMasterNPCPlugin.java
package plugin.npc;

import com.rs.game.entity.actor.npc.NPC;
import com.rs.game.entity.actor.player.Player;
import com.rs.game.plugin.type.NPCPlugin;
import com.rs.utility.game.ClickOption;

/**
 * @author Tyluur <itstyluur@gmail.com>
 * @since 8/31/2017
 */
public class SlayerMasterNPCPlugin extends NPCPlugin {
	
	@Override
	public void handle(Player player, NPC npc, ClickOption option) {
		player.getDialogueManager().startDialogue("Turael", npc.getId());
	}
	
	@Override
	public void register() {
		register(8461, ClickOption.FIRST);
	}
}

File: src/main/kotlin/plugin/npc/SlayerMasterNPCPlugin.kt
package plugin.npc

import org.redrune.game.content.plugin.type.NPCPlugin
import org.redrune.game.entity.actor.npc.NPC
import org.redrune.game.entity.actor.player.Player

/**
 * @author Tyluur <itstyluur@icloud.com>
 * @since 8/31/2017
 */
class SlayerMasterNPCPlugin : NPCPlugin {
    override fun handle(player: Player, npc: NPC, option: String): Boolean {
        when (option) {
            "Talk-to" -> {
                player.dialogueManager.startDialogue("Turael", npc.id)
                return true
            }
        }
        return false
    }

    override fun register() {
        registerNPC(8461, "Talk-to")
    }
}